### PR TITLE
Update Threema

### PR DESCRIPTION
**Pull request by fbernhart**
threema

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

File: icons/threema.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Threema icon</title><path d="M13.75496 22.24242a1.75596 1.75596 0 11-3.51192.002 1.75596 1.75596 0 013.51192-.002zm-6.33486 0a1.75646 1.75646 0 11-3.51293.002 1.75646 1.75646 0 013.51293-.002zm12.67073 0a1.75596 1.75596 0 11-3.51193.002 1.75596 1.75596 0 013.51193-.002zM7.4771 17.24752L2.3062 18.5405l1.10498-4.41991c-1.09398-1.44197-1.73296-3.17493-1.73296-5.0389C1.67822 4.0658 6.30011-.0001 12-.0001s10.32179 4.0659 10.32179 9.0818c0 5.0159-4.6219 9.08282-10.32179 9.08282a11.44976 11.44976 0 01-4.5229-.91699zm1.59497-8.91981h-.115a.46999.46999 0 00-.46999.471v4.1259c0 .26.21.471.47.471h6.08587c.26 0 .46999-.21.46999-.47v-4.1279a.46999.46999 0 00-.47-.47h-.115V7.15773A2.92694 2.92694 0 0011.998 4.2338a2.92594 2.92594 0 00-2.92494 2.92593zm4.6849 0h-3.51392V7.15973a1.75696 1.75696 0 013.51392 0v1.16998z"/></svg>
```
Comment (adamrusted) [inline icons/threema.svg:1]: LGTM! Can you get it down to 3 decimals please?

Comment (adamrusted) [inline icons/threema.svg:1]: LGTM! Any chance you can get it down to 3 decimals?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/threema.svg (hunk 1 above)
Comment (fbernhart) [reply icons/threema.svg]: @adamrusted Done 🔒

**Review by adamrusted — APPROVED** [review]
LGTM! 🔒

**Pull request merged**